### PR TITLE
Added polish translation to messages.pl.xlf

### PR DESCRIPTION
**Pull request by Cygguu**
Updated polish translation

### CODE REVIEW

File: apps/client/src/locales/messages.pl.xlf [hunk 1]
```diff
@@ -299,7 +299,7 @@
       </trans-unit>
       <trans-unit id="6948811958230386934" datatype="html">
         <source>privacy-policy</source>
-        <target state="new">privacy-policy</target>
+        <target state="translated">polityka-prywatności</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.pl.xlf:302]: `polityka-prywatnosci` (please remove the special character as it is used as part of the url)

**Review by dtslvr — CHANGES_REQUESTED** [review]
Thank you for the contribution, @Cygguu! 😎
I have added on comment.

File: apps/client/src/locales/messages.pl.xlf (hunk 1 above)
Comment (Cygguu) [reply apps/client/src/locales/messages.pl.xlf]: Ok, done

**Review by dtslvr — APPROVED** [review]
Thanks, @Cygguu!
Ready to merge now.

**Comment by dtslvr** [comment]
This PR is related to https://github.com/ghostfolio/ghostfolio/issues/3627.

**Pull request merged**